docs/setup/betamocks.md

```diff
@@ -58,16 +58,6 @@ Each service description has a `base_uri` (pulled from Settings)
 - `method:` a symbol of the http verb `:get`, `:post`, `:put`...
 - `path:` the path that combined with the base_uri makes a full URI
 - `file_path:` where to save the file (relative to betamocks' cache dir)
-```yaml
```

Comment:
Not sure if lines 61 and 62 need to be removed

Reply:
Reinstated with a new example. Also linked to settings config file above.